### PR TITLE
Add downtime to artefacts

### PR DESCRIPTION
**Pull request by vinayvinay**
www.agileplannerapp.com/boards/173808/cards/8805

As a content designer
I want a better way to indicate on transaction start pages when a service is down for planned maintenance
So that I don't have to re-edition multiple pages and push these through workflow, then repeat the process again once the downtime is over

related:
https://github.com/alphagov/publisher/pull/340
https://github.com/alphagov/govuk_content_api/pull/209
https://github.com/alphagov/frontend/pull/709

### CODE REVIEW

File: lib/govuk_content_models/test_helpers/factories.rb [hunk 1]
```diff
@@ -19,6 +19,13 @@
     disabled true
   end
 
+  factory :downtime do
+    message "This service will be unavailable between 3:00pm and 6:00pm tomorrow"
```
Comment (alext) [inline lib/govuk_content_models/test_helpers/factories.rb]: Is this a valid example of a downtime message?  It seems somewhat brittle (ie only makes sense on the day before).

**Comment by alext** [comment]
minor comment about the factory, otherwise looks good to me.

**Comment by vinayvinay** [comment]
@alext: updated to keep factory generated message as close as possible to message generated on frontend.

**Comment by vinayvinay** [comment]
@alext: moved out view logic into Publisher.

**Pull request merged**